python/src/energuide/snippets.py

```diff
@@ -80,6 +83,7 @@ def snip_house(house: element.Element) -> HouseSnippet:
         ventilation=ventilation_strings,
         water_heating=water_heating_string,
         basements=[node.to_string() for node in basements],
+        crawlspaces=[node.to_string() for node in crawlspaces],
```

Comment:
Instead of adding both basements and crawlspaces to the raw data, what about a single entry for all basement-style nodes?

Reply:
The choice comes down to if we consider crawlspaces and slabs to be "basement-style". They end up in the same table in the report, and they are structured similarly, but not enough that they would be parsed by the same code.

It would work fine, but it'd essentially make it so `Basement.from_data(...)` does a check for what kind of basement it is parsing, and passes the node down 1 of 3 code paths.

Does that sound better to to you than doing them separately?

Reply:
I have no strong opinions on this. Go with your best judgement.

Reply:
I don't either. I guess I'll just see how it looks when I get started on crawlspaces and slabs, and see if I want to integrate them into what I've done for basements.